src/server/graphql/models/Invitation/invitationMutation.js

```diff
@@ -0,0 +1,45 @@
+import r from '../../../database/rethinkDriver';
+import {
+  GraphQLNonNull,
+  GraphQLBoolean,
+  GraphQLID,
+  GraphQLList
+} from 'graphql';
+import {Invitee} from './invitationSchema';
+import {errorObj} from '../utils';
+import {requireSUOrTeamMember, getUserId} from '../authorization';
+
+import {resolveSentEmails, makeInvitations, getTeamNameInvitedBy, sendInvitations} from './helpers';
+
+export default {
+  inviteTeamMembers: {
+    type: GraphQLBoolean,
+    description: 'Send invitation emails to a list of email addresses, add them to the invitation table',
+    args: {
+      teamId: {
+        type: new GraphQLNonNull(GraphQLID),
+        description: 'The id of the inviting team'
+      },
+      invitees: {
+        type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(Invitee)))
+      }
+    },
+    async resolve(source, {invitees, teamId}, {authToken}) {
```

Comment:
invitations are tricky... PS i haven't tested this, we can do some tests after the UI for it is up.